apps/ai-content-generator/src/components/app/dialog/common-generator/field-selector/content-source/ContentSource.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, FormControl, Select } from '@contentful/f36-components';
+
+interface Props {
+  title: string;
+  isNewText: boolean;
+  onChange: () => void;
+  selectFieldWidth: number;
+}
+
+function ContentSource(props: Props) {
+  const { title, isNewText, onChange, selectFieldWidth } = props;
+
+  return (
+    <Box css={{ width: selectFieldWidth }}>
```

Comment:
What do you think about moving the html into a generic component? That way we're not duplicating html between this and the Entry field list